### PR TITLE
Release v2.6.1

### PR DESCRIPTION
**Pull request by AishwaryaNanna**
- Fixes a compilation error with iOS app inbox callback method `messageDidSelect`.

### CODE REVIEW

File: Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist [hunk 1]
```diff
@@ -50,11 +50,11 @@
 		<key>NSAllowsArbitraryLoads</key>
 		<true/>
 	</dict>
-	<key>CleverTapRegion</key>
-	<string></string>
 	<key>CleverTapAccountID</key>
 	<string>TEST-R78-ZZK-955Z</string>
 	<key>CleverTapToken</key>
 	<string>TEST-311-ba2</string>
+	<key>CleverTapRegion</key>
```
Comment (darshanclevertap) [inline Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist:57]: @AishwaryaNanna Why is this key empty?

**Review by darshanclevertap — COMMENTED** [review]
Please resolve the comments

File: Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist (hunk 1 above)
Comment (AishwaryaNanna) [reply Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist]: Its updated with latest ios sdk version update

Comment (akashvercetti) [reply Samples/Cordova/ExampleProject/platforms/ios/ExampleProject/ExampleProject-Info.plist]: removed the optional region key.

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**